getting_started/administrators.adoc

```diff
@@ -66,6 +66,12 @@ $ sudo docker run -d --name "origin" \
         openshift/origin start
 ----
 +
+NOTE: If you want to use
+../install_config/aggregate_logging.html[origin
```

Comment:
Sorry, I should have clarified this will need a `link:` at the start of it since it's an [internal xref](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#internal-cross-references):

```
link:../install_config/aggregate_logging.html[origin aggregated logging]
```